### PR TITLE
Hide Links to Coverage If Cancer Track

### PR DESCRIPTION
**Pull request by mikaell**
This PR adds a functionality or fixes a bug: hides links to coverage reports if the scout case is configured for _cancer_.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data2. 

Find one cancer and one not-cancer case. Both should display links to coverage on the main branch.

image

**Expected outcome**:
Apply this patch. Afterwards the cancer-case will not display links to coverage. The rare disease case will show links to coverage. 

The functionality should be working.

image

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by mikaell** [comment]
I think this is ready for review.

File: scout/server/blueprints/cases/templates/cases/gene_panel.html [hunk 1]
```diff
@@ -91,7 +91,7 @@ <h6 class="ms-3 mt-3"><span class="fas fa-clipboard-list"></span>&nbsp;HPO gene
         {% endif %}
         )
         {% if case.dynamic_gene_list_edited %} <span class="text-danger ml-3"> (edited) </span> {% endif %}
-        {% if config.SQLALCHEMY_DATABASE_URI and case.dynamic_gene_list|length > 0 %}
+        {% if config.SQLALCHEMY_DATABASE_URI and case.dynamic_gene_list|length > 0 and case.track != 'cancer' %}
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/gene_panel.html:94]:
```suggestion
        {% if case.track != 'cancer' and config.SQLALCHEMY_DATABASE_URI and case.dynamic_gene_list|length > 0 %}
```

**Review by northwestwitch — APPROVED** [review]
Looks good! I have feeling that if you put case.track != 'cancer' in front of the other 2 conditions the evaluation of the ifs might be a bit faster.

**Comment by mikaell** [comment]
OK to merge?

**Comment by northwestwitch** [comment]
> OK to merge?

Yes! 💯

**Pull request merged**